fast64_internal/oot/oot_level_writer.py

```diff
@@ -270,6 +259,7 @@ def readSceneData(scene, scene_properties, sceneHeader, alternateSceneHeaders):
     scene.skyboxID = getCustomProperty(sceneHeader, "skyboxID")
     scene.skyboxCloudiness = getCustomProperty(sceneHeader, "skyboxCloudiness")
     scene.skyboxLighting = getCustomProperty(sceneHeader, "skyboxLighting")
+    scene.lightMode = sceneHeader.skyboxLighting
```

Comment:
I think I mentioned this before, but I don't know why this is being re-added; `lightMode` is the same property as `skyboxLighting` except it doesn't handle the custom option correctly.

Reply:
yea sorry, I was just confused by the new enum for a moment and forgot to remove this lol